napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -79,7 +79,6 @@ def test_notification_manager_via_gui(
         ]:
             notification_manager.records = []
             qtbot.mouseClick(btt, Qt.LeftButton)
-            qtbot.wait(500)
```

Comment:
@andy-sweet I got 500 from here.

Reply:
Oh I see. I interpreted this as a hopeful wait (i.e. hoping that the thread had finished after this duration) rather than a timeout (i.e. the thread has taken too long, we should stop waiting). So that's why I used the longer 5 seconds as the default timeout, which is also consistent with `QtBot`. Does that make sense?

Reply:
yes